Reports/router.js

```diff
@@ -1,4 +1,38 @@
 var express = require('express');
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017');
```

Comment:
I think if we put this line in one of the higher-level files (e.g. server.js), then all other references to mongoose would be connected.  I.e., this line would only be called once, and URL:PORT would only be needed where this line is called.